src/renderer/components/layout/setting-layout.scss

```diff
@@ -136,13 +136,9 @@
       width: 100%;
       max-width: 740px;
       min-width: 460px;
-      padding: 60px 40px 80px;
-
-      > section {
-        &:last-of-type {
-          margin-bottom: 80px;
-        }
-      }
+      padding: 60px 40px 0;
+      height: max-content;
```

Comment:
`max-content` value works with flexboxes too? 
I was thinking it's applicable to `display: grid` children only?..

Reply:
No, it applicable to any `width/height` properties without restrictions. https://developer.mozilla.org/en-US/docs/Web/CSS/max-content